tools/ipapy2/ipapy2_MS1_annotation.xml

```diff
@@ -62,6 +62,11 @@
         Annotation of the dataset based on the MS1 information. Prior probabilities
         are based on mass only, while post probabilities are based on mass, RT,
         previous knowledge and isotope patterns.
+
+        https://github.com/francescodc87/ipaPy2#usage
+
+        https://academic.oup.com/bioinformatics/article/39/7/btad455/7230779
```

Comment:
please use the citation annotations for this. Freetext is very unstructured. If you add this to structured elements Galaxy can do some cool stuff with it.

Reply:
There is already a citations section in the help text